### PR TITLE
basic code example and links to Spectrum1D.write

### PR DESCRIPTION
**Pull request by kecnry**
This PR adds a very brief example of how to use `Spectrum1D.write` and links to the API docs within the main spectrum1d landing page on the docs.

See [rendered docs from this PR](https://specutils--1017.org.readthedocs.build/en/1017/spectrum1d.html#writing-to-a-file)

### CODE REVIEW

File: docs/spectrum1d.rst [hunk 1]
```diff
@@ -87,6 +87,18 @@ A full list of the supported formats is shown in the table below.
 
 | More information on creating custom loaders can be found in the :doc:`custom loading </custom_loading>` page.
 
+Writing to a File
+-----------------
+
+Similarly, a `~specutils.Spectrum1D` object can be saved to any file format supported by
+:class:`~astropy.nddata.CCDData` via the :meth:`specutils.Spectrum1D.write` method.
```
Comment (pllim) [inline docs/spectrum1d.rst:94]: Why is CCDData involved here?

https://github.com/astropy/specutils/blob/25cb121f8da53a7e79ebc10b94f665e28e0399d4/specutils/spectra/spectrum1d.py#L20

File: docs/spectrum1d.rst [hunk 2]
```diff
@@ -87,6 +87,18 @@ A full list of the supported formats is shown in the table below.
...
+
+.. code-block:: python
+
+    >>> from specutils import Spectrum1D
+    >>> spec1d = Spectrum1D.read("/path/to/input.fits")  # doctest: +SKIP
```
Comment (pllim) [inline docs/spectrum1d.rst:99]: Do we really need this? Reading is already covered in https://specutils.readthedocs.io/en/stable/spectrum1d.html#reading-from-a-file

```suggestion
```

**Comment by pllim** [comment]
Maybe also want to link to https://specutils.readthedocs.io/en/stable/custom_loading.html#creating-a-custom-writer (using ref)?

File: docs/spectrum1d.rst (hunk 1 above)
Comment (kecnry) [reply docs/spectrum1d.rst:94]: It comes from [NDIOMixin](https://docs.astropy.org/en/stable/api/astropy.nddata.NDIOMixin.html) whose API docs state CCDData 🤷

File: docs/spectrum1d.rst (hunk 2 above)
Comment (kecnry) [reply docs/spectrum1d.rst:99]: Should the snippets be somewhat standalone or is it ok to assume at this point that the user already has a `spectrum1D` object loaded/created?

Comment (pllim) [reply docs/spectrum1d.rst:99]: Depends on who you ask. 😆 

You can `ref` back to the "reading" section if you want, but I think it is overkill to tell people how to read again.

File: docs/spectrum1d.rst (hunk 1 above)
Comment (pllim) [reply docs/spectrum1d.rst:94]: I think NDIOMixin used CCDData as an example of implementation using the mixin. I don't think specutils has anything to do with CCDData.

Comment (kecnry) [reply docs/spectrum1d.rst:94]: ok, would you rather that I just point directly to NDIOMixin then, or nothing at all and just leave the link to the API docs which inherits from NDIOMixin API?

Comment (pllim) [reply docs/spectrum1d.rst:94]: Why not link to https://specutils.readthedocs.io/en/latest/spectrum1d.html#reading-from-a-file (with `ref`)?

Comment (kecnry) [reply docs/spectrum1d.rst]: that is _immediately_ before this sub-section, so I'll just update the text to mention "supported formats".

File: docs/spectrum1d.rst [hunk 3]
```diff
@@ -87,6 +87,18 @@ A full list of the supported formats is shown in the table below.
...
+Writing to a File
+-----------------
+
+Similarly, a `~specutils.Spectrum1D` object can be saved to any of the supported formats using the
+:meth:`specutils.Spectrum1D.write` method.
+
+.. code-block:: python
+
+    >>> spec1d.write("/path/to/output.fits")  # doctest: +SKIP
+
```
Comment (pllim) [inline docs/spectrum1d.rst:99]:
```suggestion
```

File: docs/custom_loading.rst [hunk 4]
```diff
@@ -143,6 +143,8 @@ many `~specutils.Spectrum1D` objects. This method is available since
 `astropy.io.registry.read`).
 
 
+.. _custom_writer
```
Comment (pllim) [inline docs/custom_loading.rst:146]:
```suggestion
.. _custom_writer:
```

**Review by pllim — APPROVED** [review]
LGTM now. Thanks!

No need to wait for CI. This just touch doc without doctest.

**Pull request merged**